meetings/working-groups/extensions/Extension-API-docs.md

```diff
@@ -5,9 +5,9 @@ Extensions introduce new requirements for our API reference pipeline. The additi
 - The docs term is "Extension Methods". That term currently means "extension methods with an instance receiver". Now, extensions can be properties, indexers, or operators. These extension members can be accessed as either an instance member on the extended type, or as a static member on the extended type.
 - Readers need to know if the receiver is an instance of a type, or the type itself.
 - Readers occasionally need to know the class name of holding the extension, typically for disambiguation.
-- The extension container becomes an unspeakable name, and it should be elided in documentation.
+- The extension block isn't emitted as part of the output. All extension members become members of the containing static class. An additional "skeleton" nested class (described later in the document) provides the organization for extension blocks.
```

Comment:
> The extension block isn't emitted as part of the output
I didn't understand that statement.

I'd say:
The extension block is emitted with skeleton members and xml docs. It is given an unspeakable name.
In addition, implementation methods are emitted in the containing static class. Those get an `inheritdoc` as documentation.